### PR TITLE
151755521 make gaming functional

### PR DESCRIPTION
**Pull request by gbengaoyetade**
#### What does this PR do?
Makes the gaming functionality work

#### Description of Task to be completed?
As a user, after I opt to start a or join a game, I should be able to effectively participate and carry out required game actions
#### How should this be manually tested?
Get at least 3 tabs open and then click start game
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#151755521
#### Screenshots (if appropriate)
screen shot 2017-10-08 at 12 37 29 pm

#### Questions:
None

### CODE REVIEW

**Review by nebanat — APPROVED** [review]

File: public/views/answers.html [hunk 1]
```diff
@@ -42,7 +43,7 @@
           <h3>How To Play</h3>
         </div>
         <div class="container">
-          <ol id="oh-el" class="game-instructions">
+          <ol id="oh-el" class="justify">
```
Comment (segunolalive) [inline public/views/answers.html]: Why did you change game instructions class

File: public/views/answers.html [hunk 2]
```diff
@@ -31,7 +43,7 @@
           <h3>How To Play</h3>
         </div>
         <div class="container">
-          <ol id="oh-el" class="game-instructions">
```
Comment (segunolalive) [inline public/views/answers.html:46]: why did you remove this class

**Review by segunolalive — CHANGES_REQUESTED** [review]

File: public/views/answers.html (hunk 1 above)
Comment (gbengaoyetade) [reply public/views/answers.html]: The styling didn't look go enough. I got a suggestion from a team mate and that suggestion was what I implemented.

File: public/views/answers.html (hunk 2 above)
Comment (gbengaoyetade) [reply public/views/answers.html:46]: I removed it to because I changed the gaming screen style. My changes are not so much.

**Review by ibonly — APPROVED** [review]
